contracts/arbitrum/connectors/aave/v3/main.sol

```diff
@@ -316,6 +404,32 @@ abstract contract AaveResolver is Events, Helpers {
 		_eventName = "LogSetUserEMode(uint8)";
 		_eventParam = abi.encode(categoryId);
 	}
+
+	/**
+	 * @dev Approve Delegation
+	 * @notice Gives approval to delegate debt tokens
+	 * @param token The address of token
+	 * @param rateMode The type of borrow debt
+	 * @param delegateTo The address to whom the user is delegating
+	 * @param amount The amount 
+	 */
+	function approveDelegation(address token, , uint256 amount, uint256 rateMode, address delegateTo)
+		external
+		payable
+		returns (string memory _eventName, bytes memory _eventParam)
+	{
+		require(rateMode == 1 || rateMode == 2, "Invalid debt type");
+
+		bool isEth = token == ethAddr;
+		address _token = isEth ? wethAddr : token;
+
+		address _dToken = getDTokenAddr(_token, rateMode);
+		DTokenInterface(_dToken).approveDelegation(delegateTo, amount);
+
+		_eventName = "LogApproveDelegation(address,uint16,address,uint256)";
+		_eventParam = abi.encode(token, rateMode, delegateTo, amount);
+
+	}
 }
 
 contract ConnectV2AaveV3Arbitrum is AaveResolver {
```

Comment:
<img width="408" alt="image" src="https://user-images.githubusercontent.com/22830915/165531704-98072249-9719-45db-a55b-7e0067de2275.png">
Need to change to `AaveV3-v1.2`

Reply:
Got it